manifests/e/EaseUS/EaseUSTodoPCTrans/13.2/EaseUS.EaseUSTodoPCTrans.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: EaseUS.EaseUSTodoPCTrans
+PackageVersion: 13.2
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  Scope: machine

```

Comment:
As you asked in the discussion - If this only installs in machine scope, this is appropriate. If it can also be installed just for the current user, best to leave it out.

Reply:
I've installed it again manually and I didn't see any option to install for current user. Besides it installed on Public Desktop instead of user desktop